src/tooltip/tooltip.js

```diff
@@ -302,6 +299,18 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
 
             attrs.$observe( prefix+'Title', function ( val ) {
               ttScope.title = val;
+              if (prefix === 'popover') {
```

Comment:
Why does the `popover` prefix need special handling? If a user used the `$tooltip` service to create their own tooltip directive, they wouldn't be able to get the special behavior that `popover` has unless they use the same prefix.